deploy/jenkins/jobs/Provision/jobs/dev/jobs/KnowledgePlatform/jobs/Search/config.xml

```diff
@@ -95,7 +95,7 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
       <configVersion>2</configVersion>
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```

Comment:
@SMY-ALTAMASH Why is this pointing to devops repo? All KP should be in KP

Reply:
good catch will change it